host/xtest/regression_1000.c

```diff
@@ -2349,3 +2349,214 @@ static void xtest_tee_test_1032(ADBG_Case_t *c)
 }
 ADBG_CASE_DEFINE(regression, 1032, xtest_tee_test_1032,
 		"Register read-only shared memory");
+
+/*
+ * regression_1033
+ * Depends on kernel commit "tee: add support for application-based session login methods"
+ * Mailing list archives: https://lists.trustedfirmware.org/pipermail/op-tee/2020-October/000211.html
+ * Upstream: <put sha-1 here when known>
+ *
+ * xtest skips the test when not built with OpenSSL.
+ */
+static void xtest_tee_test_1033(ADBG_Case_t *c)
+{
+#ifdef OPENSSL_FOUND
+	TEEC_Result result = TEEC_ERROR_GENERIC;
+	uint32_t ret_orig = 0;
+	TEEC_Session session = { };
+	uint32_t login = UINT32_MAX;
+	TEEC_UUID client_uuid = { };
+	TEEC_UUID expected_client_uuid = { };
+	TEEC_UUID uuid_ns = { };
+	char uuid_name[TEE_UUID_NS_NAME_SIZE + 4] = { };
```

Comment:
Since you really mean `PATH_MAX + 4` why not use that instead and leave `TEE_UUID_NS_NAME_SIZE` alone?

Reply:
Because the kernel patch changes `TEE_UUID_NS_NAME_SIZE` to `PATH_MAX` and that would introduce a discrepancy.

Reply:
I'm sorry, I don't follow.

Reply:
https://lists.trustedfirmware.org/pipermail/op-tee/2020-October/000211.html includes the following lines:
```
-#define TEE_UUID_NS_NAME_SIZE	128
+#define TEE_UUID_NS_NAME_SIZE	PATH_MAX
```
I could leave `TEE_UUID_NS_NAME_SIZE` at 128 in the tests, while the kernel patch changes it to `PATH_MAX`, but I'm not sure that's right.

Reply:
OK, a comment somewhere in the function explaining that we truncate the buffer at the same place as the kernel would help.